src/app.js

```diff
@@ -19,7 +18,6 @@ app.use('/api/v1/items', routes);
 app.use('/api/v1/users', emailAuthRoutes);
 app.use('/api/v1/users/auth', socialAuthRoutes);
 app.use('/api/v1/users/profile', profileRouter);
-app.use('/api/v1/items/search', searchRouter);
```

Comment:
Hey @SimonImanigirimpuhwe , You have removed the line that enable search items,

Reply:
It was removed unintentionally, resolved issue to be committed asap